frappe/public/js/frappe/form/controls/datetime.js

```diff
@@ -35,19 +35,23 @@ frappe.ui.form.ControlDatetime = class ControlDatetime extends frappe.ui.form.Co
 	get_now_date() {
 		return frappe.datetime.now_datetime(true);
 	}
+	/** @param {string | null} value */
 	parse(value) {
-		if (value) {
-			value = frappe.datetime.user_to_str(value, false);
+		let parsed = typeof value === "string" ? value.trim() : "";
```

Comment:
Same as in ControlDate. Can’t we inherit a new sanitization helper from the parent class?